components/image/images.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Images | GlexUI</title>
+    <link rel="stylesheet" href="../../style.css">
+</head>
+<body>
+        <div class="img-responsive-container">
+            <img class="img-responsive" src="https://picsum.photos/1000/600" alt="">
+        </div>
+        <div class="img-container">
+            <img class="img-semi-square" src="https://picsum.photos/300/300" alt="">
+        </div>
+        <div class="img-container">
+            <img class="img-circle" src="https://picsum.photos/600/600" alt="">
+        </div>
```

Comment:
Always give alt (alternate) in (avatar, badge).

Reply:
done Thank for your review